osmosis-types/gamm/pool-models/balancer/pool.go

```diff
@@ -74,7 +75,7 @@ func NewBalancerPool(poolId uint64, balancerPoolParams PoolParams, assets []Pool
 // GetAddress returns the address of a pool.
 // If the pool address is not bech32 valid, it returns an empty address.
 func (p Pool) GetAddress() sdk.AccAddress {
-	addr, err := sdk.AccAddressFromBech32(p.Address)
+	_, addr, err := bech32.DecodeAndConvert(p.Address)
```

Comment:
why noy use ` func AccAddressFromBech32(address string, checkHRP string)`

I assume you this is for a osmosis address

Reply:
yup, will use the utils func.